code/game/objects/effects/decals/contraband.dm

```diff
@@ -12,8 +12,8 @@
 	var/poster_type
 	var/obj/structure/sign/poster/poster_structure
 
-/obj/item/poster/New(loc, obj/structure/sign/poster/new_poster_structure)
-	..()
+/obj/item/poster/Initialize(mapload, obj/structure/sign/poster/new_poster_structure)
+	. = ..()
```

Comment:
While this is an `/obj/item`, not an `/obj/structure`, this had to also be done since posters were converted, and this would load before its internal poster could `Initialize()`, hence the conversion. 

Note, there is not a race condition here. Any new atoms which are spawned during the initialization of SSatoms are immediately initialized before the list continues.